CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.10.0
+
+* Preparation for [HttpHeaders change]. Update signature of `set()` to match
+  new signature of `HttpHeaders`. The parameter is not yet forwarded and
```

Comment:
This description doesn't make sense here. We are changing behavior based on this.